<?xml version="1.0" encoding="UTF-8"?>
<!-- Copyright (c) all rights. http://networker.vachok.ru 2019. -->

<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="28" hostname="HOME" failures="31" tests="399" name="networker" time="384.661" errors="0" timestamp="2019-08-10T12:03:16 MSK">
    <testcase classname="ru.vachok.networker.AbstractNetworkerFactoryTest" name="getSSHFactoryOverAbsFactory" time="0.953"/>
    <testcase classname="ru.vachok.networker.AbstractNetworkerFactoryTest" name="testCreateNetMonitorFactory" time="0.507">
        <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at ru.vachok.networker.AbstractNetworkerFactoryTest.testCreateNetMonitorFactory(AbstractNetworkerFactoryTest.java:59)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testCreateNetMonitorFactory -->
    <testcase classname="ru.vachok.networker.AbstractNetworkerFactoryTest" name="testCreateSSHFactory" time="0.002"/>
    <testcase classname="ru.vachok.networker.AbstractNetworkerFactoryTest" name="testGetFilesFactory" time="0.002"/>
    <testcase classname="ru.vachok.networker.AbstractNetworkerFactoryTest" name="testGetInfoFactory" time="0.001"/>
    <testcase classname="ru.vachok.networker.AbstractNetworkerFactoryTest" name="testGetSSHFactory" time="0.498"/>
    <testcase classname="ru.vachok.networker.AbstractNetworkerFactoryTest" name="testNetScanServiceFactory" time="0.001"/>
    <testcase classname="ru.vachok.networker.IntoApplicationTest.ArgsReaderTest" name="testFillArgs" time="0.0"/>
    <testcase classname="ru.vachok.networker.IntoApplicationTest.ArgsReaderTest" name="testRealRun" time="0.0"/>
    <testcase classname="ru.vachok.networker.IntoApplicationTest.ArgsReaderTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.VisitorTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.ssh.AccessListsCheckUniqTest" name="testConnectTo" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.ssh.AccessListsCheckUniqTest" name="testRun" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.ssh.AccessListsCheckUniqTest" name="testTestToString" time="0.002"/>
    <testcase classname="ru.vachok.networker.controller.CalculateCTRLTest" name="testGetM" time="0.004"/>
    <testcase classname="ru.vachok.networker.controller.CalculateCTRLTest" name="testTimeStamp" time="0.003"/>
    <testcase classname="ru.vachok.networker.statistics.WeekComputerUserResolvedStatsTest" name="getStatsTest" time="3.375"/>
    <testcase classname="ru.vachok.networker.ad.ActDirectoryCTRLTest" name="testAdFoto" time="2.489"/>
    <testcase classname="ru.vachok.networker.ad.ActDirectoryCTRLTest" name="testAdUsersComps" time="0.17"/>
    <testcase classname="ru.vachok.networker.net.scanner.CheckerIpHTMLTest" name="testCheckIP" time="0.017"/>
    <testcase classname="ru.vachok.networker.net.scanner.CheckerIpHTMLTest" name="testToString1" time="0.009"/>
    <testcase classname="ru.vachok.networker.exe.schedule.WeekStatsTest" name="testGetInetStats" time="0.044"/>
    <testcase classname="ru.vachok.networker.exe.schedule.WeekStatsTest" name="testGetPCStats" time="2.882"/>
    <testcase classname="ru.vachok.networker.componentsrepo.systray.ActionDelTMPTest" name="testActionPerformed" time="0.007"/>
    <testcase classname="ru.vachok.networker.componentsrepo.systray.ActionDelTMPTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.exe.runnabletasks.ChkMailAndUpdateDBTest" name="testRunCheck" time="0.012"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.WhoIsWithSRVTest" name="testWhoIs" time="3.754"/>
    <testcase classname="ru.vachok.networker.componentsrepo.systray.actions.ActionMakeInfoAboutOldCommonFilesTest" name="testActionPerformed" time="0.002">
        <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at ru.vachok.networker.componentsrepo.systray.actions.ActionMakeInfoAboutOldCommonFilesTest.testActionPerformed(ActionMakeInfoAboutOldCommonFilesTest.java:34)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testActionPerformed -->
    <testcase classname="ru.vachok.networker.componentsrepo.systray.actions.ActionMakeInfoAboutOldCommonFilesTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.common.CleanerTest" name="testGetPathAttrMap" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.common.CleanerTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.systray.ActionRunTest" name="experementalAction" time="3.061"/>
    <testcase classname="ru.vachok.networker.componentsrepo.systray.ActionRunTest" name="testActionPerformed$$COPY" time="0.003"/>
    <testcase classname="ru.vachok.networker.componentsrepo.systray.ActionRunTest" name="testTestToString" time="0.005"/>
    <testcase classname="ru.vachok.networker.ad.common.CommonCTRLTest" name="testCommonGET" time="0.004"/>
    <testcase classname="ru.vachok.networker.ad.common.CommonCTRLTest" name="testCommonSearch" time="0.015"/>
    <testcase classname="ru.vachok.networker.ad.ADComputerTest" name="testToString1" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.common.CommonSRVTest" name="testReStoreDir" time="3.49"/>
    <testcase classname="ru.vachok.networker.ad.common.CommonSRVTest" name="testSearchByPat" time="2.089"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testCheckCommonRightsForUserName" time="0.04"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testFromADUsersList" time="0.209"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testGetAdComputer" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testGetAdUser" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testGetDetails" time="30.846"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testGetUserInputRaw" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testRun" time="0.011"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testSetUserInputRaw" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testShowADPCList" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testTestToString" time="0.002"/>
    <testcase classname="ru.vachok.networker.ad.ADSrvTest" name="testUserSetter" time="0.785"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.ConcreteFolderACLWriterTest" name="testRun" time="5.078"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.ConcreteFolderACLWriterTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.ConcreteFolderACLWriterTest" name="testWriteACLs" time="3.558"/>
    <testcase classname="ru.vachok.networker.ad.user.ADUserTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.pc.PCOnTest" name="testGetInfoAbout" time="0.196"/>
    <testcase classname="ru.vachok.networker.ad.pc.PCOnTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.AnketaKonfetaCRTLTest" name="testGetMapForAnketa" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.AnketaKonfetaCRTLTest" name="testPostAnketa" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.CountSizeOfWorkDirTest" name="countDiskSpace" time="0.015"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.CountSizeOfWorkDirTest" name="testCall" time="0.674"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testAdSrv" time="0.0"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testConnection" time="0.089"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testDiapazonedScanInfo" time="0.044"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testGetMailProps" time="0.001"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testGetPFLists" time="0.0"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testGetProps" time="0.0"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testGetUserPref" time="0.009"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testGetUserResolver" time="17.282"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testIpFlushDNS" time="0.033"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testLaunchRegRuFTPLibsUploader" time="0.059"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testLoadPropsAndWriteToFile" time="0.003"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testNetScannerSvc" time="0.03"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testSaveLogsToDB" time="0.417"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testScanOnline" time="0.092"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testSimpleCalculator" time="0.001"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testSshActs" time="0.0"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testTemporaryFullInternet" time="0.001"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testThreadConfig" time="0.0"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testVersionInfo" time="0.018"/>
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testVisitor" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.user.FileADUsersParserTest" name="adUsersFromFileGetter" time="0.935"/>
    <testcase classname="ru.vachok.networker.ad.user.FileADUsersParserTest" name="testFileParser" time="0.608"/>
    <testcase classname="ru.vachok.networker.ad.user.FileADUsersParserTest" name="testGetAdUsers" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.user.FileADUsersParserTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.AppInfoOnLoadTest" name="onePCMonStart" time="0.001"/>
    <testcase classname="ru.vachok.networker.AppInfoOnLoadTest" name="realRun" time="2.97"/>
    <testcase classname="ru.vachok.networker.AppInfoOnLoadTest" name="testGetBuildStamp" time="0.163"/>
    <testcase classname="ru.vachok.networker.AppInfoOnLoadTest" name="testGetIISLogSize" time="0.003"/>
    <testcase classname="ru.vachok.networker.AppInfoOnLoadTest" name="testGetThisDelay" time="0.026"/>
    <testcase classname="ru.vachok.networker.AppInfoOnLoadTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.pc.PCOffTest" name="testGetInfoAbout$$COPY" time="0.006"/>
    <testcase classname="ru.vachok.networker.ad.pc.PCOffTest" name="testGetInfoAbout" time="1.791"/>
    <testcase classname="ru.vachok.networker.ad.pc.PCOffTest" name="testSetInfo" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.pc.PCOffTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.restapi.database.DataConnectToAdapterTest" name="testGetDataSource" time="0.001"/>
    <testcase classname="ru.vachok.networker.restapi.database.DataConnectToAdapterTest" name="testGetI" time="0.0"/>
    <testcase classname="ru.vachok.networker.restapi.database.DataConnectToAdapterTest" name="testGetRegRuMysqlLibConnection" time="0.033"/>
    <testcase classname="ru.vachok.networker.restapi.database.DataConnectToAdapterTest" name="testGetSavepoint" time="0.0"/>
    <testcase classname="ru.vachok.networker.restapi.database.DataConnectToAdapterTest" name="testSetSavepoint" time="0.0"/>
    <testcase classname="ru.vachok.networker.restapi.message.DBMessengerTest" name="sendMessage" time="0.043">
        <failure type="java.lang.AssertionError"
                 message="DROP command denied to user &amp;apos;u0466446_network&amp;apos;@&amp;apos;91.192.22.201&amp;apos; for table &amp;apos;ru_vachok_networker&amp;apos;
NativeConstructorAccessorImpl.java: java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
NativeConstructorAccessorImpl.java: java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
DelegatingConstructorAccessorImpl.java: java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
Constructor.java: java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
Constructor.java: java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
Util.java: com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
Util.java: com.mysql.jdbc.Util.getInstance(Util.java:408)
SQLError.java: com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
MysqlIO.java: com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3976)
MysqlIO.java: com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3912)
MysqlIO.java: com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
MysqlIO.java: com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
ConnectionImpl.java: com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2486)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1858)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2079)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2013)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:5104)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1998)
DBMessengerTest.java: ru.vachok.networker.restapi.message.DBMessengerTest.checkMessageExistsInDatabase(DBMessengerTest.java:78)
DBMessengerTest.java: ru.vachok.networker.restapi.message.DBMessengerTest.sendMessage(DBMessengerTest.java:58)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
DelegatingMethodAccessorImpl.java: java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Method.java: java.base/java.lang.reflect.Method.invoke(Method.java:567)
MethodInvocationHelper.java: org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
TestInvoker.java: org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:570)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
MethodRunner.java: org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
TestInvoker.java: org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
TestRunner.java: org.testng.TestRunner.privateRun(TestRunner.java:763)
TestRunner.java: org.testng.TestRunner.run(TestRunner.java:594)
SuiteRunner.java: org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
SuiteRunner.java: org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
SuiteRunner.java: org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
SuiteRunner.java: org.testng.SuiteRunner.run(SuiteRunner.java:304)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
TestNG.java: org.testng.TestNG.runSuitesSequentially(TestNG.java:1146)
TestNG.java: org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
TestNG.java: org.testng.TestNG.runSuites(TestNG.java:997)
TestNG.java: org.testng.TestNG.run(TestNG.java:965)
IDEARemoteTestNG.java: org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
RemoteTestNGStarter.java: org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)



 expected [null] but found [com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: DROP command denied to user &amp;apos;u0466446_network&amp;apos;@&amp;apos;91.192.22.201&amp;apos; for table &amp;apos;ru_vachok_networker&amp;apos;]">
            <![CDATA[java.lang.AssertionError: DROP command denied to user 'u0466446_network'@'91.192.22.201' for table 'ru_vachok_networker'
Util.java: com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
Util.java: com.mysql.jdbc.Util.getInstance(Util.java:408)
SQLError.java: com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
MysqlIO.java: com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3976)
MysqlIO.java: com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3912)
MysqlIO.java: com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
MysqlIO.java: com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
ConnectionImpl.java: com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2486)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1858)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2079)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2013)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:5104)
PreparedStatement.java: com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1998)
DBMessengerTest.java: ru.vachok.networker.restapi.message.DBMessengerTest.checkMessageExistsInDatabase(DBMessengerTest.java:78)
DBMessengerTest.java: ru.vachok.networker.restapi.message.DBMessengerTest.sendMessage(DBMessengerTest.java:58)



 expected [null] but found [com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: DROP command denied to user 'u0466446_network'@'91.192.22.201' for table 'ru_vachok_networker']
at ru.vachok.networker.restapi.message.DBMessengerTest.checkMessageExistsInDatabase(DBMessengerTest.java:84)
at ru.vachok.networker.restapi.message.DBMessengerTest.sendMessage(DBMessengerTest.java:58)
... Removed 60 stack frames]]>
        </failure>
    </testcase> <!-- sendMessage -->
    <testcase classname="ru.vachok.networker.restapi.message.DBMessengerTest" name="testAsSingle" time="0.002"/>
    <testcase classname="ru.vachok.networker.restapi.message.DBMessengerTest" name="testToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.restapi.props.DBPropsCallableTest" name="localFind" time="0.195"/>
    <testcase classname="ru.vachok.networker.restapi.props.DBPropsCallableTest" name="testCall" time="0.154"/>
    <testcase classname="ru.vachok.networker.restapi.props.DBPropsCallableTest" name="testDelProps" time="0.064"/>
    <testcase classname="ru.vachok.networker.restapi.props.DBPropsCallableTest" name="testGetProps" time="0.037"/>
    <testcase classname="ru.vachok.networker.restapi.props.DBPropsCallableTest" name="testGetRegSourceForProperties" time="0.001"/>
    <testcase classname="ru.vachok.networker.restapi.props.DBPropsCallableTest" name="testSetProps" time="0.152"/>
    <testcase classname="ru.vachok.networker.restapi.props.DBPropsCallableTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.exe.schedule.DeadLockMonitorTest" name="testCall" time="0.003"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.DeleterTempTest" name="testToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.monitor.DiapazonScanTest" name="isOldFilesExistsTest" time="0.002"/>
    <testcase classname="ru.vachok.networker.net.monitor.DiapazonScanTest" name="makeFilesMapTest" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.monitor.DiapazonScanTest" name="scanFilesTest" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.DiapazonScanTest" name="testRun" time="0.072"/>
    <testcase classname="ru.vachok.networker.net.monitor.DiapazonScanTest" name="testTheInfoToString" time="0.007"/>
    <testcase classname="ru.vachok.networker.net.monitor.DiapazonScanTest" name="testToString1" time="0.043"/>
    <testcase classname="ru.vachok.networker.controller.ErrCtrTest" name="testErrHandle" time="0.001"/>
    <testcase classname="ru.vachok.networker.controller.ErrCtrTest" name="testGetErrorPath" time="0.001"/>
    <testcase classname="ru.vachok.networker.controller.ErrCtrTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.controller.ExCTRLTest" name="testExchangeWorks" time="0.001"/>
    <testcase classname="ru.vachok.networker.controller.ExCTRLTest" name="testOstPstGet" time="0.0">
        <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at ru.vachok.networker.controller.ExCTRLTest.testOstPstGet(ExCTRLTest.java:82)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testOstPstGet -->
    <testcase classname="ru.vachok.networker.controller.ExCTRLTest" name="testRuleSetGet" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.ExCTRLTest" name="testRuleSetPost" time="0.002"/>
    <testcase classname="ru.vachok.networker.controller.ExCTRLTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.ExCTRLTest" name="testUplFile" time="0.004"/>
    <testcase classname="ru.vachok.networker.net.monitor.ExecScanTest" name="cpOldFile$$COPY" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.monitor.ExecScanTest" name="oneIPScanTest" time="1.87"/>
    <testcase classname="ru.vachok.networker.net.monitor.ExecScanTest" name="testRun" time="9.005"/>
    <testcase classname="ru.vachok.networker.net.monitor.ExecScanTest" name="toStringTest" time="0.008"/>
    <testcase classname="ru.vachok.networker.ExitAppTest" name="testGetVisitsMap" time="0.001"/>
    <testcase classname="ru.vachok.networker.ExitAppTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.ExitAppTest" name="testWriteOwnObject" time="0.015"/>
    <testcase classname="ru.vachok.networker.mail.ExSRVTest" name="testFileAsStrings" time="0.173"/>
    <testcase classname="ru.vachok.networker.controller.FileCleanerCTRLTest" name="testGetFilesInfo" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.FileCleanerCTRLTest" name="testPostFile" time="0.001"/>
    <testcase classname="ru.vachok.networker.controller.FileCleanerCTRLTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.restapi.props.FilePropsLocalTest" name="testDelProps" time="0.0"/>
    <testcase classname="ru.vachok.networker.restapi.props.FilePropsLocalTest" name="testGetProps" time="0.163"/>
    <testcase classname="ru.vachok.networker.restapi.props.FilePropsLocalTest" name="testSetProps" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.common.FileRestorerTest" name="realCall" time="0.001">
        <failure type="java.lang.AssertionError" message="null
FutureTask.java: java.base/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:418)
FutureTask.java: java.base/java.util.concurrent.FutureTask.get(FutureTask.java:203)
FileRestorerTest.java: ru.vachok.networker.ad.common.FileRestorerTest.realCall(FileRestorerTest.java:45)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
DelegatingMethodAccessorImpl.java: java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Method.java: java.base/java.lang.reflect.Method.invoke(Method.java:567)
MethodInvocationHelper.java: org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
TestInvoker.java: org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:570)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
MethodRunner.java: org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
TestInvoker.java: org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
TestRunner.java: org.testng.TestRunner.privateRun(TestRunner.java:763)
TestRunner.java: org.testng.TestRunner.run(TestRunner.java:594)
SuiteRunner.java: org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
SuiteRunner.java: org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
SuiteRunner.java: org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
SuiteRunner.java: org.testng.SuiteRunner.run(SuiteRunner.java:304)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
TestNG.java: org.testng.TestNG.runSuitesSequentially(TestNG.java:1146)
TestNG.java: org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
TestNG.java: org.testng.TestNG.runSuites(TestNG.java:997)
TestNG.java: org.testng.TestNG.run(TestNG.java:965)
IDEARemoteTestNG.java: org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
RemoteTestNGStarter.java: org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)



 expected [null] but found [java.lang.InterruptedException]">
            <![CDATA[java.lang.AssertionError: null
FutureTask.java: java.base/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:418)
FutureTask.java: java.base/java.util.concurrent.FutureTask.get(FutureTask.java:203)
FileRestorerTest.java: ru.vachok.networker.ad.common.FileRestorerTest.realCall(FileRestorerTest.java:45)



 expected [null] but found [java.lang.InterruptedException]
at ru.vachok.networker.ad.common.FileRestorerTest.realCall(FileRestorerTest.java:52)
... Removed 55 stack frames]]>
        </failure>
    </testcase> <!-- realCall -->
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSearcherTest" name="makeSearchByName" time="0.079"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSearcherTest" name="makeSearchByTime" time="0.202"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testAppendObjectToFile" time="0.006"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testCopyOrDelFile" time="0.004"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testCountStringsInFile" time="5.826"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testDelTemp" time="0.48"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testError" time="0.012"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testPackFiles" time="0.029"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testReadFile" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testReadFileToList" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testReadFileToQueue" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testReadFileToSet" time="0.002"/>
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testWriteFile" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.FilesZipPackerTest" name="testCall" time="0.012"/>
    <testcase classname="ru.vachok.networker.accesscontrol.inetstats.ResolveUserInDataBaseTest" name="testGetUsage" time="15.251"/>
    <testcase classname="ru.vachok.networker.info.stats.WeeklyInternetStatsTest.InetStatSorterTest" name="testRun" time="0.001"/>
    <testcase classname="ru.vachok.networker.info.inet.PCInetTest" name="testGetUsage" time="28.977"/>
    <testcase classname="ru.vachok.networker.info.inet.PCInetTest" name="testShowLog" time="0.001"/>
    <testcase classname="ru.vachok.networker.restapi.props.InitPropertiesAdapterTest" name="testSetProps" time="0.35"/>
    <testcase classname="ru.vachok.networker.info.stats.WeeklyInternetStatsTest" name="dayOfWeekTesting" time="0.0"/>
    <testcase classname="ru.vachok.networker.info.stats.WeeklyInternetStatsTest" name="testDeleteFrom" time="0.028"/>
    <testcase classname="ru.vachok.networker.info.stats.WeeklyInternetStatsTest" name="testInetStat" time="0.001"/>
    <testcase classname="ru.vachok.networker.info.stats.WeeklyInternetStatsTest" name="testRun" time="0.001"/>
    <testcase classname="ru.vachok.networker.info.stats.WeeklyInternetStatsTest" name="testSelectFrom" time="0.029"/>
    <testcase classname="ru.vachok.networker.IntoApplicationTest" name="runMainApp" time="0.003"/>
    <testcase classname="ru.vachok.networker.IntoApplicationTest" name="testBeforeSt" time="0.158"/>
    <testcase classname="ru.vachok.networker.IntoApplicationTest" name="testMain" time="0.002"/>
    <testcase classname="ru.vachok.networker.IntoApplicationTest" name="testReloadConfigurableApplicationContext" time="0.082">
        <failure type="org.springframework.core.task.TaskRejectedException"
                 message="Executor [java.util.concurrent.ThreadPoolExecutor@cfaafc6[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]] did not accept task: AppInfoOnLoad{, thisDelay=18&amp;lt;br&amp;gt;&amp;lt;br&amp;gt;infoForU ends. now ftpUploadTask(). Result: 1565427499137&amp;lt;br&amp;gt;144 MB IIS Logs
&amp;lt;br&amp;gt;HOME&amp;lt;br&amp;gt;new AppComponents().launchRegRuFTPLibsUploader() = true&amp;lt;br&amp;gt;144 MB IIS Logs
}">
            <![CDATA[org.springframework.core.task.TaskRejectedException: Executor [java.util.concurrent.ThreadPoolExecutor@cfaafc6[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]] did not accept task: AppInfoOnLoad{, thisDelay=18<br><br>infoForU ends. now ftpUploadTask(). Result: 1565427499137<br>144 MB IIS Logs
<br>HOME<br>new AppComponents().launchRegRuFTPLibsUploader() = true<br>144 MB IIS Logs
}
at org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor.execute(ThreadPoolTaskExecutor.java:317)
at ru.vachok.networker.IntoApplication.afterSt(IntoApplication.java:109)
at ru.vachok.networker.IntoApplication.ArgsReader.readArgs(ArgsReader.java:68)
at ru.vachok.networker.IntoApplication.ArgsReader.fillArgsMap(ArgsReader.java:106)
at ru.vachok.networker.IntoApplication.ArgsReader.run(ArgsReader.java:48)
at ru.vachok.networker.IntoApplication.main(IntoApplication.java:93)
at ru.vachok.networker.IntoApplicationTest.testReloadConfigurableApplicationContext(IntoApplicationTest.java:72)
Caused by: java.util.concurrent.RejectedExecutionException: Task AppInfoOnLoad{, thisDelay=18<br><br>infoForU ends. now ftpUploadTask(). Result: 1565427499137<br>144 MB IIS Logs
<br>HOME<br>new AppComponents().launchRegRuFTPLibsUploader() = true<br>144 MB IIS Logs
} rejected from java.util.concurrent.ThreadPoolExecutor@cfaafc6[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]
at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2055)
at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:825)
at java.base/java.util.concurrent.ThreadPoolExecutor.execute(ThreadPoolExecutor.java:1355)
at org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor.execute(ThreadPoolTaskExecutor.java:314)
... 32 more
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testReloadConfigurableApplicationContext -->
    <testcase classname="ru.vachok.networker.componentsrepo.InvokeIllegalExceptionTest" name="getMyThrow" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="getExecution$$COPY" time="0.002"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="kudrMonitorTest" time="1.002"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="testGetMapOfConditionsTypeNameTypeCondition" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="testGetPingResultStr" time="0.005"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="testGetStatistics" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="testIsReach" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="testPingDevices" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="testWriteLog" time="1.002"/>
    <testcase classname="ru.vachok.networker.exe.schedule.MailIISLogsCleanerTest" name="testRun" time="0.233"/>
    <testcase classname="ru.vachok.networker.mail.testserver.MailPOPTesterTest" name="logFileReader" time="2.824"/>
    <testcase classname="ru.vachok.networker.controller.MatrixCtrTest" name="testGetFirst" time="0.036"/>
    <testcase classname="ru.vachok.networker.controller.MatrixCtrTest" name="testGetWorkPosition" time="0.048"/>
    <testcase classname="ru.vachok.networker.controller.MatrixCtrTest" name="testSetCurrentProvider" time="0.014"/>
    <testcase classname="ru.vachok.networker.controller.MatrixCtrTest" name="testShowResults" time="0.001"/>
    <testcase classname="ru.vachok.networker.accesscontrol.MatrixSRVTest" name="testSearchAccessPrincipals" time="0.039"/>
    <testcase classname="ru.vachok.networker.restapi.message.MessageToTrayTest" name="testConfirm" time="0.003"/>
    <testcase classname="ru.vachok.networker.restapi.message.MessageToTrayTest" name="testErrorAlert" time="0.001"/>
    <testcase classname="ru.vachok.networker.restapi.message.MessageToTrayTest" name="testInfo" time="0.0"/>
    <testcase classname="ru.vachok.networker.restapi.message.MessageToTrayTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.restapi.message.MessageToTrayTest" name="testWarn" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.MyCalenTest" name="planTruncateTableUsers" time="0.003">
        <failure type="java.util.concurrent.RejectedExecutionException"
                 message="Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@2979c6ef[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@137d04d8[Wrapped task = ru.vachok.networker.componentsrepo.services.MyCalen$$Lambda$664/0x0000000801847840@25b8f9d2]] rejected from java.util.concurrent.ScheduledThreadPoolExecutor@73316a0a[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]">
            <![CDATA[java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@2979c6ef[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@137d04d8[Wrapped task = ru.vachok.networker.componentsrepo.services.MyCalen$$Lambda$664/0x0000000801847840@25b8f9d2]] rejected from java.util.concurrent.ScheduledThreadPoolExecutor@73316a0a[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]
at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2055)
at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:825)
at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.scheduleWithFixedDelay(ScheduledThreadPoolExecutor.java:680)
at ru.vachok.networker.componentsrepo.services.MyCalen.planTruncateTableUsers(MyCalen.java:170)
at ru.vachok.networker.componentsrepo.services.MyCalenTest.planTruncateTableUsers(MyCalenTest.java:72)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- planTruncateTableUsers -->
    <testcase classname="ru.vachok.networker.componentsrepo.services.MyCalenTest" name="testGetNextDay" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.MyCalenTest" name="testGetNextDayofWeek" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.MyCalenTest" name="testGetThisDay" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.MyCalenTest" name="testGetTimeInfo" time="0.001">
        <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at ru.vachok.networker.componentsrepo.services.MyCalenTest.testGetTimeInfo(MyCalenTest.java:32)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testGetTimeInfo -->
    <testcase classname="ru.vachok.networker.componentsrepo.services.MyCalenTest" name="testToStringS" time="0.001"/>
    <testcase classname="ru.vachok.networker.accesscontrol.NameOrIPCheckerTest" name="testCheckPat" time="0.005"/>
    <testcase classname="ru.vachok.networker.accesscontrol.NameOrIPCheckerTest" name="testResolveIP" time="0.27"/>
    <testcase classname="ru.vachok.networker.abstr.NetKeeperTest" name="testGetAllDevices" time="0.0">
        <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at ru.vachok.networker.abstr.NetKeeperTest.testGetAllDevices(NetKeeperTest.java:66)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testGetAllDevices -->
    <testcase classname="ru.vachok.networker.abstr.NetKeeperTest" name="testGetCurrentScanFiles" time="0.001"/>
    <testcase classname="ru.vachok.networker.abstr.NetKeeperTest" name="testGetCurrentScanLists" time="0.003">
        <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at ru.vachok.networker.abstr.NetKeeperTest.testGetCurrentScanLists(NetKeeperTest.java:43)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testGetCurrentScanLists -->
    <testcase classname="ru.vachok.networker.abstr.NetKeeperTest" name="testGetKudrWorkTime" time="0.0">
        <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at ru.vachok.networker.abstr.NetKeeperTest.testGetKudrWorkTime(NetKeeperTest.java:72)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testGetKudrWorkTime -->
    <testcase classname="ru.vachok.networker.abstr.NetKeeperTest" name="testGetNetworkPCs" time="0.001"/>
    <testcase classname="ru.vachok.networker.abstr.NetKeeperTest" name="testGetOnePcMonitor" time="0.0"/>
    <testcase classname="ru.vachok.networker.abstr.NetKeeperTest" name="testGetScanFiles" time="0.0"/>
    <testcase classname="ru.vachok.networker.abstr.NetKeeperTest" name="testMakeFilesMap" time="0.01"/>
    <testcase classname="ru.vachok.networker.componentsrepo.data.NetListsTest" name="testEditOffLines" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.data.NetListsTest" name="testGetI" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.data.NetListsTest" name="testGetMapAddr" time="0.195"/>
    <testcase classname="ru.vachok.networker.componentsrepo.data.NetListsTest" name="testGetOffLines" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.data.NetListsTest" name="testGetOnLinesResolve" time="0.022"/>
    <testcase classname="ru.vachok.networker.componentsrepo.data.NetListsTest" name="testSetOffLines" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.data.NetListsTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.NetMonitorPTVTests" name="ptvIfBigTest" time="0.012"/>
    <testcase classname="ru.vachok.networker.net.monitor.NetMonitorPTVTests" name="testRun" time="2.703"/>
    <testcase classname="ru.vachok.networker.net.monitor.NetMonitorPTVTests" name="testToString1" time="2.985"/>
    <testcase classname="ru.vachok.networker.net.scanner.NetScanCtrTest" name="testNetScan" time="0.006"/>
    <testcase classname="ru.vachok.networker.net.scanner.NetScanCtrTest" name="testPcNameForInfo" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.scanner.NetScanCtrTest" name="testPingAddr" time="0.002"/>
    <testcase classname="ru.vachok.networker.net.scanner.NetScanCtrTest" name="testPingPost" time="0.005"/>
    <testcase classname="ru.vachok.networker.net.scanner.PcNamesScannerTest" name="testCheckMapSizeAndDoAction" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.scanner.PcNamesScannerTest" name="testGetThePc" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.scanner.PcNamesScannerTest" name="testSetThePc" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.scanner.PcNamesScannerTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.scanner.PcNamesScannerTest" name="testTheSETOfPCNamesPref" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.scanner.PcNamesScannerTest" name="testTheSETOfPcNames" time="0.001"/>
    <testcase classname="ru.vachok.networker.abstr.monitors.NetScanServiceTest" name="testGetExecution" time="0.002"/>
    <testcase classname="ru.vachok.networker.abstr.monitors.NetScanServiceTest" name="testGetMonitoringRunnable" time="0.001"/>
    <testcase classname="ru.vachok.networker.abstr.monitors.NetScanServiceTest" name="testGetPingResultStr" time="0.0"/>
    <testcase classname="ru.vachok.networker.abstr.monitors.NetScanServiceTest" name="testGetStatistics" time="0.001"/>
    <testcase classname="ru.vachok.networker.abstr.monitors.NetScanServiceTest" name="testIsReach" time="0.001"/>
    <testcase classname="ru.vachok.networker.abstr.monitors.NetScanServiceTest" name="testPingDevices" time="0.042"/>
    <testcase classname="ru.vachok.networker.abstr.monitors.NetScanServiceTest" name="testWriteLogToFile" time="0.001"/>
    <testcase classname="ru.vachok.networker.controller.NoHupOutTest" name="testNoHupGet" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.NoHupOutTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.OdinAssSyncCTRLTest" name="testResetForm" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.OdinAssSyncCTRLTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.OdinAssSyncCTRLTest" name="testUploadFiles" time="0.002"/>
    <testcase classname="ru.vachok.networker.controller.OdinAssSyncCTRLTest" name="testViewPage" time="0.001"/>
    <testcase classname="ru.vachok.networker.controller.OkCTRLTest" name="testOkStr" time="0.001"/>
    <testcase classname="ru.vachok.networker.controller.ConfigsReloaderTest" name="testMakeOk" time="0.004">
        <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at ru.vachok.networker.controller.ConfigsReloaderTest.testMakeOk(OKMakerTest.java:43)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testMakeOk -->
    <testcase classname="ru.vachok.networker.controller.ConfigsReloaderTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.common.OldBigFilesInfoCollectorTest" name="testCall" time="0.004"/>
    <testcase classname="ru.vachok.networker.ad.common.OldBigFilesInfoCollectorTest" name="testGetDate" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.common.OldBigFilesInfoCollectorTest" name="testGetStartPath" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.common.OldBigFilesInfoCollectorTest" name="testSetDate" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.common.OldBigFilesInfoCollectorTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.OwnerFixerTest" name="testRun" time="1.681"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.OwnerFixerTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.PageGenerationHelperTest" name="testTestToString" time="0.003"/>
    <testcase classname="ru.vachok.networker.info.PageGenerationHelperTest" name="testGetInfoAbout" time="0.001"/>
    <testcase classname="ru.vachok.networker.info.PageGenerationHelperTest" name="testSetInfo" time="0.006"/>
    <testcase classname="ru.vachok.networker.info.PageGenerationHelperTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.PCMonitoringTest" name="exp$$WriteLog" time="0.003"/>
    <testcase classname="ru.vachok.networker.net.monitor.PCMonitoringTest" name="testGetExecution" time="0.826"/>
    <testcase classname="ru.vachok.networker.net.monitor.PCMonitoringTest" name="testGetMonitoringRunnable" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.PCMonitoringTest" name="testGetPingResultStr" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.monitor.PCMonitoringTest" name="testGetStatistics" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.PCMonitoringTest" name="testIsReach" time="0.002"/>
    <testcase classname="ru.vachok.networker.net.monitor.PCMonitoringTest" name="testRun" time="0.002">
        <failure type="java.lang.AssertionError" message="null
FutureTask.java: java.base/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:418)
FutureTask.java: java.base/java.util.concurrent.FutureTask.get(FutureTask.java:203)
PCMonitoringTest.java: ru.vachok.networker.net.monitor.PCMonitoringTest.testRun(PCMonitoringTest.java:102)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
DelegatingMethodAccessorImpl.java: java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Method.java: java.base/java.lang.reflect.Method.invoke(Method.java:567)
MethodInvocationHelper.java: org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
TestInvoker.java: org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:570)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
MethodRunner.java: org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
TestInvoker.java: org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
TestRunner.java: org.testng.TestRunner.privateRun(TestRunner.java:763)
TestRunner.java: org.testng.TestRunner.run(TestRunner.java:594)
SuiteRunner.java: org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
SuiteRunner.java: org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
SuiteRunner.java: org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
SuiteRunner.java: org.testng.SuiteRunner.run(SuiteRunner.java:304)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
TestNG.java: org.testng.TestNG.runSuitesSequentially(TestNG.java:1146)
TestNG.java: org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
TestNG.java: org.testng.TestNG.runSuites(TestNG.java:997)
TestNG.java: org.testng.TestNG.run(TestNG.java:965)
IDEARemoteTestNG.java: org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
RemoteTestNGStarter.java: org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)



 expected [null] but found [java.lang.InterruptedException]">
            <![CDATA[java.lang.AssertionError: null
FutureTask.java: java.base/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:418)
FutureTask.java: java.base/java.util.concurrent.FutureTask.get(FutureTask.java:203)
PCMonitoringTest.java: ru.vachok.networker.net.monitor.PCMonitoringTest.testRun(PCMonitoringTest.java:102)



 expected [null] but found [java.lang.InterruptedException]
at ru.vachok.networker.net.monitor.PCMonitoringTest.testRun(PCMonitoringTest.java:108)
... Removed 55 stack frames]]>
        </failure>
    </testcase> <!-- testRun -->
    <testcase classname="ru.vachok.networker.net.monitor.PCMonitoringTest" name="testWriteLog" time="0.001"/>
    <testcase classname="ru.vachok.networker.info.stats.ComputerUserResolvedStatsTest" name="testCall" time="4.172"/>
    <testcase classname="ru.vachok.networker.info.stats.ComputerUserResolvedStatsTest" name="testSelectFrom" time="2.058"/>
    <testcase classname="ru.vachok.networker.ad.PCUserResolverTest" name="testRun" time="0.671"/>
    <testcase classname="ru.vachok.networker.net.ssh.PfListsTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.PhotoConverterSRVTest" name="convertFoto" time="0.31"/>
    <testcase classname="ru.vachok.networker.ad.PhotoConverterSRVTest" name="testGetAdFotoFile" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.PhotoConverterSRVTest" name="testPsCommands" time="0.002"/>
    <testcase classname="ru.vachok.networker.ad.PhotoConverterSRVTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.monitor.PingerFromFileTest" name="testIsReach" time="0.009"/>
    <testcase classname="ru.vachok.networker.net.monitor.PingerFromFileTest" name="testPingDev" time="0.005"/>
    <testcase classname="ru.vachok.networker.net.monitor.PingerFromFileTest" name="testRun" time="10.628"/>
    <testcase classname="ru.vachok.networker.restapi.props.PreferencesHelperTest" name="testDelProps$$COPY" time="0.027"/>
    <testcase classname="ru.vachok.networker.restapi.props.PreferencesHelperTest" name="testDelProps" time="0.004">
        <failure type="java.lang.AssertionError" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.delProps(PreferencesHelper.java:29)

TODO: 06.08.2019 (23:25)
PreferencesHelper.java: ru.vachok.networker.restapi.props.PreferencesHelper.delProps(PreferencesHelper.java:29)
PreferencesHelperTest.java: ru.vachok.networker.restapi.props.PreferencesHelperTest.testDelProps(PreferencesHelperTest.java:72)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
DelegatingMethodAccessorImpl.java: java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Method.java: java.base/java.lang.reflect.Method.invoke(Method.java:567)
MethodInvocationHelper.java: org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
TestInvoker.java: org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:570)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
MethodRunner.java: org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
TestInvoker.java: org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
TestRunner.java: org.testng.TestRunner.privateRun(TestRunner.java:763)
TestRunner.java: org.testng.TestRunner.run(TestRunner.java:594)
SuiteRunner.java: org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
SuiteRunner.java: org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
SuiteRunner.java: org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
SuiteRunner.java: org.testng.SuiteRunner.run(SuiteRunner.java:304)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
TestNG.java: org.testng.TestNG.runSuitesSequentially(TestNG.java:1146)
TestNG.java: org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
TestNG.java: org.testng.TestNG.runSuites(TestNG.java:997)
TestNG.java: org.testng.TestNG.run(TestNG.java:965)
IDEARemoteTestNG.java: org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
RemoteTestNGStarter.java: org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)



 expected [null] but found [ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.delProps(PreferencesHelper.java:29)

TODO: 06.08.2019 (23:25)]">
            <![CDATA[java.lang.AssertionError: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.delProps(PreferencesHelper.java:29)

TODO: 06.08.2019 (23:25)
PreferencesHelper.java: ru.vachok.networker.restapi.props.PreferencesHelper.delProps(PreferencesHelper.java:29)
PreferencesHelperTest.java: ru.vachok.networker.restapi.props.PreferencesHelperTest.testDelProps(PreferencesHelperTest.java:72)



 expected [null] but found [ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.delProps(PreferencesHelper.java:29)

TODO: 06.08.2019 (23:25)]
at ru.vachok.networker.restapi.props.PreferencesHelperTest.testDelProps(PreferencesHelperTest.java:75)
... Removed 55 stack frames]]>
        </failure>
    </testcase> <!-- testDelProps -->
    <testcase classname="ru.vachok.networker.restapi.props.PreferencesHelperTest" name="testGetProps" time="0.001">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.getProps(PreferencesHelper.java:19)

TODO: 06.08.2019 (23:25)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.getProps(PreferencesHelper.java:19)

TODO: 06.08.2019 (23:25)
at ru.vachok.networker.restapi.props.PreferencesHelper.getProps(PreferencesHelper.java:19)
at ru.vachok.networker.restapi.props.PreferencesHelperTest.testGetProps(PreferencesHelperTest.java:48)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetProps -->
    <testcase classname="ru.vachok.networker.restapi.props.PreferencesHelperTest" name="testSetProps" time="0.004">
        <failure type="java.lang.AssertionError" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.setProps(PreferencesHelper.java:24)

TODO: 06.08.2019 (23:25)
PreferencesHelper.java: ru.vachok.networker.restapi.props.PreferencesHelper.setProps(PreferencesHelper.java:24)
PreferencesHelperTest.java: ru.vachok.networker.restapi.props.PreferencesHelperTest.testSetProps(PreferencesHelperTest.java:62)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
DelegatingMethodAccessorImpl.java: java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Method.java: java.base/java.lang.reflect.Method.invoke(Method.java:567)
MethodInvocationHelper.java: org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
TestInvoker.java: org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:570)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
MethodRunner.java: org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
TestInvoker.java: org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
TestRunner.java: org.testng.TestRunner.privateRun(TestRunner.java:763)
TestRunner.java: org.testng.TestRunner.run(TestRunner.java:594)
SuiteRunner.java: org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
SuiteRunner.java: org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
SuiteRunner.java: org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
SuiteRunner.java: org.testng.SuiteRunner.run(SuiteRunner.java:304)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
TestNG.java: org.testng.TestNG.runSuitesSequentially(TestNG.java:1146)
TestNG.java: org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
TestNG.java: org.testng.TestNG.runSuites(TestNG.java:997)
TestNG.java: org.testng.TestNG.run(TestNG.java:965)
IDEARemoteTestNG.java: org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
RemoteTestNGStarter.java: org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)



 expected [null] but found [ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.setProps(PreferencesHelper.java:24)

TODO: 06.08.2019 (23:25)]">
            <![CDATA[java.lang.AssertionError: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.setProps(PreferencesHelper.java:24)

TODO: 06.08.2019 (23:25)
PreferencesHelper.java: ru.vachok.networker.restapi.props.PreferencesHelper.setProps(PreferencesHelper.java:24)
PreferencesHelperTest.java: ru.vachok.networker.restapi.props.PreferencesHelperTest.testSetProps(PreferencesHelperTest.java:62)



 expected [null] but found [ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.restapi.props.PreferencesHelper.setProps(PreferencesHelper.java:24)

TODO: 06.08.2019 (23:25)]
at ru.vachok.networker.restapi.props.PreferencesHelperTest.testSetProps(PreferencesHelperTest.java:65)
... Removed 55 stack frames]]>
        </failure>
    </testcase> <!-- testSetProps -->
    <testcase classname="ru.vachok.networker.componentsrepo.exceptions.PropertiesAppNotFoundExceptionTest" name="testThrow" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.libswork.RegRuDBLibsTest" name="testConnectTo" time="0.014"/>
    <testcase classname="ru.vachok.networker.net.libswork.RegRuDBLibsTest" name="testGetContentsQueue" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.RegRuFTPLibsUploaderTest" name="checkPassOverDB on RegRuFTPLibsUploader{ftpClient=null}" time="0.045"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.RegRuFTPLibsUploaderTest" name="chkPC$$COPY on RegRuFTPLibsUploader{ftpClient=null}" time="0.004"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.RegRuFTPLibsUploaderTest" name="continueConnect on RegRuFTPLibsUploader{ftpClient=null}" time="0.019"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.RegRuFTPLibsUploaderTest" name="ftpTest on RegRuFTPLibsUploader{ftpClient=null}" time="4.963"/>
    <testcase classname="ru.vachok.networker.restapi.database.RegRuMysqlLocTest" name="testGetDefaultConnection" time="0.035"/>
    <testcase classname="ru.vachok.networker.restapi.database.RegRuMysqlLocTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.RightsCheckerTest" name="copyExistsFiles" time="0.0">
        <failure type="java.lang.AssertionError" message="G:\My_Proj\FtpClientPlus\modules\networker\logscommon\common.own did not expect to find [true] but found [false]">
            <![CDATA[java.lang.AssertionError: G:\My_Proj\FtpClientPlus\modules\networker\logscommon\common.own did not expect to find [true] but found [false]
at ru.vachok.networker.ad.usermanagement.RightsCheckerTest.copyExistsFiles(RightsCheckerTest.java:82)
... Removed 29 stack frames]]>
        </failure>
    </testcase> <!-- copyExistsFiles -->
    <testcase classname="ru.vachok.networker.ad.usermanagement.RightsCheckerTest" name="runChecker" time="0.0">
        <failure type="java.lang.AssertionError" message="null
FutureTask.java: java.base/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:418)
FutureTask.java: java.base/java.util.concurrent.FutureTask.get(FutureTask.java:203)
RightsCheckerTest.java: ru.vachok.networker.ad.usermanagement.RightsCheckerTest.runChecker(RightsCheckerTest.java:61)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
DelegatingMethodAccessorImpl.java: java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Method.java: java.base/java.lang.reflect.Method.invoke(Method.java:567)
MethodInvocationHelper.java: org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
TestInvoker.java: org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:570)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
MethodRunner.java: org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
TestInvoker.java: org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
TestRunner.java: org.testng.TestRunner.privateRun(TestRunner.java:763)
TestRunner.java: org.testng.TestRunner.run(TestRunner.java:594)
SuiteRunner.java: org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
SuiteRunner.java: org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
SuiteRunner.java: org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
SuiteRunner.java: org.testng.SuiteRunner.run(SuiteRunner.java:304)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
TestNG.java: org.testng.TestNG.runSuitesSequentially(TestNG.java:1146)
TestNG.java: org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
TestNG.java: org.testng.TestNG.runSuites(TestNG.java:997)
TestNG.java: org.testng.TestNG.run(TestNG.java:965)
IDEARemoteTestNG.java: org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
RemoteTestNGStarter.java: org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)



 expected [null] but found [java.lang.InterruptedException]">
            <![CDATA[java.lang.AssertionError: null
FutureTask.java: java.base/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:418)
FutureTask.java: java.base/java.util.concurrent.FutureTask.get(FutureTask.java:203)
RightsCheckerTest.java: ru.vachok.networker.ad.usermanagement.RightsCheckerTest.runChecker(RightsCheckerTest.java:61)



 expected [null] but found [java.lang.InterruptedException]
at ru.vachok.networker.ad.usermanagement.RightsCheckerTest.runChecker(RightsCheckerTest.java:64)
... Removed 55 stack frames]]>
        </failure>
    </testcase> <!-- runChecker -->
    <testcase classname="ru.vachok.networker.ad.usermanagement.ACLParserTest" name="readUserACL" time="1.319"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.ACLParserTest" name="realRunTest" time="3.025"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.ACLParserTest" name="testFoundPatternMap" time="5.311"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.ACLParserTest" name="testGetSearchPatterns" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.ACLParserTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.exe.runnabletasks.external.SaveLogsToDBTest" name="testGetI" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="fileOnToLastCopyTest" time="0.006"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetExecution" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetFileMAXOnlines" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetMonitoringRunnable" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetOnlinesFile" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetPingResultStr" time="0.003"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetReplaceFileNamePattern" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetStatistics" time="0.002"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetStatistics" time="0.002"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetStatistics" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetStatistics" time="0.002"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetStatistics" time="0.002"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testGetTimeToEndStr" time="0.0"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testIsReach" time="0.035"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testPingDevices" time="0.219"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testRun" time="0.008"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testScanOnlineLastBigger" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testToString1" time="0.056"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testToStringTester" time="0.045"/>
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="testWriteLogToFile" time="0.003"/>
    <testcase classname="ru.vachok.networker.sysinfo.ServiceInfoCtrlTest" name="testCloseApp" time="0.002"/>
    <testcase classname="ru.vachok.networker.sysinfo.ServiceInfoCtrlTest" name="testInfoMapping" time="0.001"/>
    <testcase classname="ru.vachok.networker.sysinfo.ServiceInfoCtrlTest" name="testInfoMappingCOPY" time="0.002"/>
    <testcase classname="ru.vachok.networker.sysinfo.ServiceInfoCtrlTest" name="testOffPC" time="0.002"/>
    <testcase classname="ru.vachok.networker.sysinfo.ServiceInfoCtrlTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.controller.ShowAllDevCTRLTest" name="testModel" time="0.068"/>
    <testcase classname="ru.vachok.networker.controller.ShowAllDevCTRLTest" name="testModel" time="0.046"/>
    <testcase classname="ru.vachok.networker.controller.ShowAllDevCTRLTest" name="testModel" time="0.044"/>
    <testcase classname="ru.vachok.networker.controller.ShowAllDevCTRLTest" name="testModel" time="0.04"/>
    <testcase classname="ru.vachok.networker.controller.ShowAllDevCTRLTest" name="testModel" time="0.052"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.SimpleCalculatorTest" name="countThreads" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.SimpleCalculatorTest" name="testCountDoubles" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.SimpleCalculatorTest" name="testGetQuery" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.SimpleCalculatorTest" name="testGetStampFromDate" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.SimpleCalculatorTest" name="testSetQuery" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.SimpleCalculatorTest" name="testToString1" time="0.0"/>
    <testcase classname="ru.vachok.networker.exe.runnabletasks.SpeedCheckerTest" name="testCall" time="2.931"/>
    <testcase classname="ru.vachok.networker.exe.runnabletasks.SpeedCheckerTest" name="testRun" time="10.117"/>
    <testcase classname="ru.vachok.networker.controller.SshActsCTRLTest" name="testAllowPOST" time="0.422"/>
    <testcase classname="ru.vachok.networker.controller.SshActsCTRLTest" name="testDelDomPOST" time="0.46"/>
    <testcase classname="ru.vachok.networker.controller.SshActsCTRLTest" name="testParseReq" time="0.002"/>
    <testcase classname="ru.vachok.networker.controller.SshActsCTRLTest" name="testSshActsGET" time="0.003"/>
    <testcase classname="ru.vachok.networker.controller.SshActsCTRLTest" name="testSshActsPOST" time="0.002"/>
    <testcase classname="ru.vachok.networker.controller.SshActsCTRLTest" name="testTempFullInetAccess" time="0.379"/>
    <testcase classname="ru.vachok.networker.controller.SshActsCTRLTest" name="testWhatSrvNeed" time="0.001"/>
    <testcase classname="ru.vachok.networker.net.ssh.SshActsTest" name="testAllowDomainAdd" time="0.605"/>
    <testcase classname="ru.vachok.networker.net.ssh.SshActsTest" name="testAllowDomainDel" time="0.625"/>
    <testcase classname="ru.vachok.networker.net.ssh.SshActsTest" name="testWhatSrvNeed" time="0.001"/>
    <testcase classname="ru.vachok.networker.SSHFactoryTest" name="testCall" time="0.465"/>
    <testcase classname="ru.vachok.networker.SSHFactoryTest" name="testDirectCall" time="0.412"/>
    <testcase classname="ru.vachok.networker.SSHFactoryTest" name="testTestToString" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.server.TelnetServerTest" name="testGetSocket" time="0.0">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.server.TelnetServerTest.testGetSocket(TelnetServerTest.java:41)

TODO: 05.08.2019 (23:43)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.server.TelnetServerTest.testGetSocket(TelnetServerTest.java:41)

TODO: 05.08.2019 (23:43)
at ru.vachok.networker.componentsrepo.server.TelnetServerTest.testGetSocket(TelnetServerTest.java:41)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetSocket -->
    <testcase classname="ru.vachok.networker.componentsrepo.server.TelnetServerTest" name="testRunSocket" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.server.TelnetServerTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.componentsrepo.server.TelnetStarterTest" name="startServer" time="0.003">
        <failure type="java.lang.AssertionError" message="null
FutureTask.java: java.base/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:418)
FutureTask.java: java.base/java.util.concurrent.FutureTask.get(FutureTask.java:203)
TelnetStarterTest.java: ru.vachok.networker.componentsrepo.server.TelnetStarterTest.startServer(TelnetStarterTest.java:46)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
NativeMethodAccessorImpl.java: java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
DelegatingMethodAccessorImpl.java: java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Method.java: java.base/java.lang.reflect.Method.invoke(Method.java:567)
MethodInvocationHelper.java: org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
TestInvoker.java: org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:570)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
MethodRunner.java: org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
TestInvoker.java: org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
TestInvoker.java: org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
TestMethodWorker.java: org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
TestRunner.java: org.testng.TestRunner.privateRun(TestRunner.java:763)
TestRunner.java: org.testng.TestRunner.run(TestRunner.java:594)
SuiteRunner.java: org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
SuiteRunner.java: org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
SuiteRunner.java: org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
SuiteRunner.java: org.testng.SuiteRunner.run(SuiteRunner.java:304)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
SuiteRunnerWorker.java: org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
TestNG.java: org.testng.TestNG.runSuitesSequentially(TestNG.java:1146)
TestNG.java: org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
TestNG.java: org.testng.TestNG.runSuites(TestNG.java:997)
TestNG.java: org.testng.TestNG.run(TestNG.java:965)
IDEARemoteTestNG.java: org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
RemoteTestNGStarter.java: org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)



 expected [null] but found [java.lang.InterruptedException]">
            <![CDATA[java.lang.AssertionError: null
FutureTask.java: java.base/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:418)
FutureTask.java: java.base/java.util.concurrent.FutureTask.get(FutureTask.java:203)
TelnetStarterTest.java: ru.vachok.networker.componentsrepo.server.TelnetStarterTest.startServer(TelnetStarterTest.java:46)



 expected [null] but found [java.lang.InterruptedException]
at ru.vachok.networker.componentsrepo.server.TelnetStarterTest.startServer(TelnetStarterTest.java:49)
... Removed 55 stack frames]]>
        </failure>
    </testcase> <!-- startServer -->
    <testcase classname="ru.vachok.networker.net.ssh.TemporaryFullInternetTest" name="sshChecker$$Copy" time="0.575"/>
    <testcase classname="ru.vachok.networker.net.ssh.TemporaryFullInternetTest" name="testRunAdd" time="0.953"/>
    <testcase classname="ru.vachok.networker.net.ssh.TemporaryFullInternetTest" name="testRunCheck" time="0.047"/>
    <testcase classname="ru.vachok.networker.net.ssh.TemporaryFullInternetTest" name="testRunFromMonitors" time="0.018"/>
    <testcase classname="ru.vachok.networker.exe.ThreadConfigTest" name="testDumpToFile" time="0.033"/>
    <testcase classname="ru.vachok.networker.exe.ThreadConfigTest" name="testExecByThreadConfig" time="0.001"/>
    <testcase classname="ru.vachok.networker.exe.ThreadConfigTest" name="testGetI" time="0.0"/>
    <testcase classname="ru.vachok.networker.exe.ThreadConfigTest" name="testGetTaskExecutor" time="0.001"/>
    <testcase classname="ru.vachok.networker.exe.ThreadConfigTest" name="testGetTaskScheduler" time="0.001"/>
    <testcase classname="ru.vachok.networker.exe.ThreadConfigTest" name="testKillAll" time="0.0"/>
    <testcase classname="ru.vachok.networker.exe.ThreadConfigTest" name="testThrNameSet" time="0.0"/>
    <testcase classname="ru.vachok.networker.exe.ThreadConfigTest" name="testToString1" time="0.002"/>
    <testcase classname="ru.vachok.networker.componentsrepo.services.TimeCheckerTest" name="testCall" time="0.011"/>
    <testcase classname="ru.vachok.networker.net.ssh.TraceroutingTest" name="testCall" time="0.005"/>
    <testcase classname="ru.vachok.networker.ad.pc.TvPcInformationTest" name="testGetInfoAbout" time="0.117"/>
    <testcase classname="ru.vachok.networker.ad.pc.TvPcInformationTest" name="testGetOnline" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.pc.TvPcInformationTest" name="testGetRunningInformation" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.pc.TvPcInformationTest" name="testGetUserFromDB" time="31.592"/>
    <testcase classname="ru.vachok.networker.ad.pc.TvPcInformationTest" name="testSetInfo" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.pc.TvPcInformationTest" name="testToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.restapi.fsworks.UpakFilesTest" name="makeZip" time="0.124"/>
    <testcase classname="ru.vachok.networker.restapi.fsworks.UpakFilesTest" name="testUpak" time="16.839"/>
    <testcase classname="ru.vachok.networker.restapi.fsworks.UpakFilesTest" name="toStringTest" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testGetAtomicTime" time="0.0">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetAtomicTime(UsefulUtilitiesTest.java:82)

TODO: 10.08.2019 (11:50)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetAtomicTime(UsefulUtilitiesTest.java:82)

TODO: 10.08.2019 (11:50)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetAtomicTime(UsefulUtilitiesTest.java:82)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetAtomicTime -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testGetDelay" time="0.001">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetDelay(UsefulUtilitiesTest.java:67)

TODO: 10.08.2019 (11:50)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetDelay(UsefulUtilitiesTest.java:67)

TODO: 10.08.2019 (11:50)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetDelay(UsefulUtilitiesTest.java:67)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetDelay -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testGetHTMLCenterRed" time="0.001"/>
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testGetMailRules" time="0.001">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetMailRules(UsefulUtilitiesTest.java:37)

TODO: 10.08.2019 (11:51)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetMailRules(UsefulUtilitiesTest.java:37)

TODO: 10.08.2019 (11:51)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetMailRules(UsefulUtilitiesTest.java:37)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetMailRules -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testGetMyTime" time="0.0">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetMyTime(UsefulUtilitiesTest.java:62)

TODO: 10.08.2019 (11:50)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetMyTime(UsefulUtilitiesTest.java:62)

TODO: 10.08.2019 (11:50)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetMyTime(UsefulUtilitiesTest.java:62)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetMyTime -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testGetStringsVisit" time="0.0">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetStringsVisit(UsefulUtilitiesTest.java:47)

TODO: 10.08.2019 (11:51)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetStringsVisit(UsefulUtilitiesTest.java:47)

TODO: 10.08.2019 (11:51)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetStringsVisit(UsefulUtilitiesTest.java:47)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetStringsVisit -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testGetUpTime" time="0.0">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetUpTime(UsefulUtilitiesTest.java:77)

TODO: 10.08.2019 (11:50)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetUpTime(UsefulUtilitiesTest.java:77)

TODO: 10.08.2019 (11:50)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetUpTime(UsefulUtilitiesTest.java:77)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetUpTime -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testGetVis" time="0.001">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetVis(UsefulUtilitiesTest.java:57)

TODO: 10.08.2019 (11:50)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetVis(UsefulUtilitiesTest.java:57)

TODO: 10.08.2019 (11:50)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testGetVis(UsefulUtilitiesTest.java:57)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testGetVis -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testIsPingOK" time="0.001">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testIsPingOK(UsefulUtilitiesTest.java:42)

TODO: 10.08.2019 (11:51)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testIsPingOK(UsefulUtilitiesTest.java:42)

TODO: 10.08.2019 (11:51)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testIsPingOK(UsefulUtilitiesTest.java:42)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testIsPingOK -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testStartTelnet" time="0.0">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testStartTelnet(UsefulUtilitiesTest.java:72)

TODO: 10.08.2019 (11:50)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testStartTelnet(UsefulUtilitiesTest.java:72)

TODO: 10.08.2019 (11:50)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testStartTelnet(UsefulUtilitiesTest.java:72)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testStartTelnet -->
    <testcase classname="ru.vachok.networker.componentsrepo.UsefulUtilitiesTest" name="testThisPC" time="0.001">
        <failure type="ru.vachok.networker.componentsrepo.exceptions.TODOException" message="This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testThisPC(UsefulUtilitiesTest.java:52)

TODO: 10.08.2019 (11:50)">
            <![CDATA[ru.vachok.networker.componentsrepo.exceptions.TODOException: This functional is not ready yet. this is ru_vachok_networker- :ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testThisPC(UsefulUtilitiesTest.java:52)

TODO: 10.08.2019 (11:50)
at ru.vachok.networker.componentsrepo.UsefulUtilitiesTest.testThisPC(UsefulUtilitiesTest.java:52)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- testThisPC -->
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLAdderTest" name="booleanAddTest" time="1.191"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLDeleterTest" name="testDeleter" time="14.977"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLManagerImplTest" name="addAccess" time="14.756"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLManagerImplTest" name="removeAccess" time="14.14"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLManagerImplTest" name="testCreateACLForUserFromExistsACL" time="0.003">
        <failure type="java.lang.AssertionError"
                 message="expected [HOME\14350:READ_DATA/WRITE_DATA/APPEND_DATA/READ_NAMED_ATTRS/WRITE_NAMED_ATTRS/EXECUTE/DELETE_CHILD/READ_ATTRIBUTES/WRITE_ATTRIBUTES/DELETE/READ_ACL/WRITE_ACL/WRITE_OWNER/SYNCHRONIZE:FILE_INHERIT/DIRECTORY_INHERIT:ALLOW] but found [HOME\14350:READ_DATA/WRITE_DATA/APPEND_DATA/READ_NAMED_ATTRS/WRITE_NAMED_ATTRS/EXECUTE/DELETE_CHILD/READ_ATTRIBUTES/WRITE_ATTRIBUTES/DELETE/READ_ACL/WRITE_ACL/WRITE_OWNER/SYNCHRONIZE:ALLOW]">
            <![CDATA[java.lang.AssertionError: expected [HOME\14350:READ_DATA/WRITE_DATA/APPEND_DATA/READ_NAMED_ATTRS/WRITE_NAMED_ATTRS/EXECUTE/DELETE_CHILD/READ_ATTRIBUTES/WRITE_ATTRIBUTES/DELETE/READ_ACL/WRITE_ACL/WRITE_OWNER/SYNCHRONIZE:FILE_INHERIT/DIRECTORY_INHERIT:ALLOW] but found [HOME\14350:READ_DATA/WRITE_DATA/APPEND_DATA/READ_NAMED_ATTRS/WRITE_NAMED_ATTRS/EXECUTE/DELETE_CHILD/READ_ATTRIBUTES/WRITE_ATTRIBUTES/DELETE/READ_ACL/WRITE_ACL/WRITE_OWNER/SYNCHRONIZE:ALLOW]
at ru.vachok.networker.ad.usermanagement.UserACLManagerImplTest.testCreateACLForUserFromExistsACL(UserACLManagerImplTest.java:105)
... Removed 30 stack frames]]>
        </failure>
    </testcase> <!-- testCreateACLForUserFromExistsACL -->
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLManagerImplTest" name="testCreateNewACL" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLManagerImplTest" name="testReplaceUsers" time="25.62"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLReplacerTest" name="testRun" time="41.599"/>
    <testcase classname="ru.vachok.networker.ad.usermanagement.UserACLReplacerTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.user.UsersKeeperTest" name="testGet24hrsTempInetList" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.user.UsersKeeperTest" name="testGetInetUniqMap" time="0.001"/>
    <testcase classname="ru.vachok.networker.ad.UserWebCTRLTest" name="testAdSrvForUser" time="0.004"/>
    <testcase classname="ru.vachok.networker.ad.UserWebCTRLTest" name="testTestToString" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.UserWebCTRLTest" name="testUserGet" time="0.0"/>
    <testcase classname="ru.vachok.networker.ad.UserWebCTRLTest" name="testUserPost" time="0.002"/>
    <testcase classname="ru.vachok.networker.mail.testserver.MailPOPTesterTest" name="testTestInput" time="0.0">
        <ignored/>
    </testcase> <!-- testTestInput -->
    <testcase classname="ru.vachok.networker.mail.testserver.MailPOPTesterTest" name="realTesting" time="0.0">
        <ignored/>
    </testcase> <!-- realTesting -->
    <testcase classname="ru.vachok.networker.mail.testserver.MailPOPTesterTest" name="testTestComplex" time="0.0">
        <ignored/>
    </testcase> <!-- testTestComplex -->
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testUpdateProps" time="0.0">
        <ignored/>
    </testcase> <!-- testUpdateProps -->
    <testcase classname="ru.vachok.networker.componentsrepo.VersionInfoTest" name="testSetParams" time="0.0">
        <ignored/>
    </testcase> <!-- testSetParams -->
    <testcase classname="ru.vachok.networker.net.monitor.ExecScanTest" name="realExecScanTest" time="0.0">
        <ignored/>
    </testcase> <!-- realExecScanTest -->
    <testcase classname="ru.vachok.networker.ad.common.FileRestorerTest" name="call" time="0.0">
        <ignored/>
    </testcase> <!-- call -->
    <testcase classname="ru.vachok.networker.ad.common.CommonCTRLTest" name="testCommonArchPOST" time="0.0">
        <ignored/>
    </testcase> <!-- testCommonArchPOST -->
    <testcase classname="ru.vachok.networker.AppInfoOnLoadTest" name="testKudrMonitor" time="0.0">
        <ignored/>
    </testcase> <!-- testKudrMonitor -->
    <testcase classname="ru.vachok.networker.AppComponentsTest" name="testConfigurableApplicationContext" time="0.0">
        <ignored/>
    </testcase> <!-- testConfigurableApplicationContext -->
    <testcase classname="ru.vachok.networker.accesscontrol.inetstats.ResolveUserInDataBaseTest" name="testShowLog" time="0.0">
        <ignored/>
    </testcase> <!-- testShowLog -->
    <testcase classname="ru.vachok.networker.ExitAppTest" name="testRun" time="0.0">
        <ignored/>
    </testcase> <!-- testRun -->
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="testGetExecution" time="0.0">
        <ignored/>
    </testcase> <!-- testGetExecution -->
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.DeleterTempTest" name="testRun" time="0.0">
        <ignored/>
    </testcase> <!-- testRun -->
    <testcase classname="ru.vachok.networker.exe.runnabletasks.external.SaveLogsToDBTest" name="testShowInfo" time="0.0">
        <ignored/>
    </testcase> <!-- testShowInfo -->
    <testcase classname="ru.vachok.networker.AppInfoOnLoadTest" name="testRun" time="0.0">
        <ignored/>
    </testcase> <!-- testRun -->
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="countStringsInFileAsStream" time="0.0">
        <ignored/>
    </testcase> <!-- countStringsInFileAsStream -->
    <testcase classname="ru.vachok.networker.exe.runnabletasks.external.SaveLogsToDBTest" name="testRun" time="0.0">
        <ignored/>
    </testcase> <!-- testRun -->
    <testcase classname="ru.vachok.networker.IntoApplicationTest" name="testGetConfigurableApplicationContext" time="0.0">
        <ignored/>
    </testcase> <!-- testGetConfigurableApplicationContext -->
    <testcase classname="ru.vachok.networker.net.scanner.ScanOnlineTest" name="offlineNotEmptTEST" time="0.0">
        <ignored/>
    </testcase> <!-- offlineNotEmptTEST -->
    <testcase classname="ru.vachok.networker.componentsrepo.fileworks.FileSystemWorkerTest" name="testCopyOrDelFileWithPath" time="0.0">
        <ignored/>
    </testcase> <!-- testCopyOrDelFileWithPath -->
    <testcase classname="ru.vachok.networker.net.monitor.KudrWorkTimeTest" name="monitorAddress$$COPY" time="0.0">
        <ignored/>
    </testcase> <!-- monitorAddress$$COPY -->
    <testcase classname="ru.vachok.networker.mail.testserver.MailPOPTesterTest" name="testTestOutput" time="0.0">
        <ignored/>
    </testcase> <!-- testTestOutput -->
    <testcase classname="ru.vachok.networker.ad.common.CleanerTest" name="testCall" time="0.0">
        <ignored/>
    </testcase> <!-- testCall -->
    <testcase classname="ru.vachok.networker.info.stats.WeeklyInternetStatsTest.InetStatSorterTest" name="sortFilesTESTCopy" time="0.0">
        <ignored/>
    </testcase> <!-- sortFilesTESTCopy -->
    <testcase classname="ru.vachok.networker.componentsrepo.VersionInfoTest" name="getParamsTEST" time="0.0">
        <ignored/>
    </testcase> <!-- getParamsTEST -->
    <testcase classname="ru.vachok.networker.exe.runnabletasks.external.SaveLogsToDBTest" name="testStartScheduled" time="0.0">
        <ignored/>
    </testcase> <!-- testStartScheduled -->
    <testcase classname="ru.vachok.networker.SSHFactoryTest" name="testOverABSFactory" time="0.0">
        <ignored/>
    </testcase> <!-- testOverABSFactory -->
</testsuite> <!-- networker -->
